apps/server/src/tasks/tasks.service.ts

```diff
@@ -172,11 +169,30 @@ export class TasksService {
     });
   }
 
-  async updateConstTask(user: User, updateTaskInput: TaskInput) {
+  async update(id: number, updateTaskInput: TaskInput) {
+    const task: Task = await Task.findOne({
+      relations: ['chunks', 'chunkInfo', 'chunkInfo.repeat'],
+      where: {
+        id: id,
+      },
+    });
+
+    task.isDone = updateTaskInput.isDone;
+
+    await task.save();
+
+    return task;
+  }
```

Comment:
right now this method in only used for changing isDone property but it shouldn't look like that anyway - changed it from default 
```
await Task.update(updateTaskInput.id, updateTaskInput);
```
because of error which occured after calling it via updateTaskMutation:
```
Error: No entity column "chillTime" was found.
    at new ApolloError (index.js?41c3:29:1)
    at eval (QueryManager.js?1cfe:95:1)
    at both (asyncMap.js?76e4:16:46)
    at eval (asyncMap.js?76e4:9:57)
    at new Promise (<anonymous>)
    at Object.then (asyncMap.js?76e4:9:1)
    at Object.eval [as next] (asyncMap.js?76e4:17:1)
    at notifySubscription (module.js?d813:132:1)
    at onNotify (module.js?d813:176:1)
    at SubscriptionObserver.next (module.js?d813:225:1)
    at eval (iteration.js?9dc9:4:50)
    at Array.forEach (<anonymous>)
    at iterateObserversSafely (iteration.js?9dc9:4:1)
    at Object.next (Concast.js?71d7:25:43)
    at notifySubscription (module.js?d813:132:1)
    at onNotify (module.js?d813:176:1)
    at SubscriptionObserver.next (module.js?d813:225:1)
    at eval (parseAndCheckHttpResponse.js?81b1:123:1)
```
found on stack that `Using save method solve No entity column [...] was found.`
https://stackoverflow.com/questions/54904349/typeorm-one-to-many-entitycolumnnotfound-no-entity-column-context-physicalp

Reply:
Hard disagree, we need to change all suitable props there.

Reply:
The problem is that we need explicitly to provide needed props there, to create the needed object. That's the purpose of update endpoint to be general. But it's not blocking, and we can omit that for now.